contributors/devel/pull-requests.md

```diff
@@ -1,72 +1,101 @@
+- Sign the CLA (prerequisite)
+- Make the PR
+- Update the `release-note-label-needed` label
+- Pass all e2e tests
+- Get a `LGTM` from a reviewer
 
-There are several requirements for the submit-queue to work:
-* Author must have signed CLA ("cla: yes" label added to PR)
-* No changes can be made since last lgtm label was applied
-* k8s-bot must have reported the GCE E2E build and test steps passed (Jenkins unit/integration, Jenkins e2e)
+## Update the Release Note Label (and Write Release Notes if Needed)
 
-Additionally, for infrequent or new contributors, we require the on call to apply the "ok-to-merge" label manually.  This is gated by the [whitelist](https://github.com/kubernetes/contrib/blob/master/mungegithub/whitelist.txt).
+Add release notes to any PR with user-visible changes, such as bug-fixes, feature additions, and output format changes.
 
-## Before sending a pull request
+The `release-note-label-needed` label is added to your PR automatically when you create it. There are a few ways to update it.
 
-The following will save time for both you and your reviewer:
+**Labels**
 
-* Enable [pre-commit hooks](development.md#committing-changes-to-your-fork) and verify they pass.
-* Verify `make verify` passes.
-* Verify `make test` passes.
-* Verify `make test-integration` passes.
+1. All pull requests are initiated with a `release-note-label-needed` label
+1. Remove the `release-note-label-needed` label and replace it with one of the other `release-note-*` labels:
+    1. `release-note-none` is a valid option if the PR does not need to be mentioned at release time
+    1. `release-note` labeled PRs generate a release note using the PR title by default OR the release-note block in the PR template, if it's filled in
 
-## Release Notes
+**Comments**
 
-This section applies only to pull requests on the master branch.
-For cherry-pick PRs, see the [Cherrypick instructions](cherry-picks.md)
+Or, commenting either `/release-note` or `/release-note-none` will also set the `release-note` or `release-note-none` labels respectively.
 
-1. All pull requests are initiated with a `release-note-label-needed` label.
-1. For a PR to be ready to merge, the `release-note-label-needed` label must be removed and one of the other `release-note-*` labels must be added.
-1. `release-note-none` is a valid option if the PR does not need to be mentioned
- at release time.
-1. `release-note` labeled PRs generate a release note using the PR title by
-   default OR the release-note block in the PR template if filled in.
-  * See the [PR template](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md) for more
-    details.
-  * PR titles and body comments are mutable and can be modified at any time
-    prior to the release to reflect a release note friendly message.
+**Descriptions**
```

Comment:
This section, might be better moved higher in the release notes section, as it's easiest for new contributors to do this as part of the PR template